### ISSUE TITLE
What is the network structure in the box？

### ISSUE THREAD

**Issue by Gamkiu** [EndlessSora/TSIT#4]
Hello, I have some questions about the paper. What is the network structure in the box(between the two Content ResBlk)? I didn't find the descriptions in your paper? And what you meaning of using Spectral Norm for all layers in the network? Is not Batch normalization and Instance normalization?

**Comment by EndlessSora** [comment]
1. Please refer to Appendix C for additional implementation details.
2. Spectral Norm is a weight normalization strategy proposed in [Spectral Normalization for Generative Adversarial Networks](https://arxiv.org/abs/1802.05957) to stabilize the GAN training. It is applied to the conv layers in our work.
